src/main/java/com/microsoft/util/CommentHelper.java

```diff
@@ -0,0 +1,89 @@
+package com.microsoft.util;
+
+import com.sun.source.doctree.DocTree;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.lang.model.element.Element;
+
+public class CommentHelper {
+    public Element element;
+    public List<? extends DocTree> inlineTags = Collections.emptyList();
+    private Utils utils;
+    private boolean hasInheritDocTag = false;
+
+    public CommentHelper(Element element, Utils utils) {
+        this.element = element;
+        this.utils = utils;
+        this.inlineTags = utils.getFullBody(element);
+        this.hasInheritDocTag = utils.hasInlineTag(inlineTags, DocTree.Kind.INHERIT_DOC);
+    }
+
+    public CommentHelper(Element element, Utils utils, List<? extends DocTree> inlineTags) {
+        this.element = element;
+        this.utils = utils;
+        this.inlineTags = inlineTags;
+        this.hasInheritDocTag = utils.hasInlineTag(inlineTags, DocTree.Kind.INHERIT_DOC);
+    }
+
+    /**
+     * Returns true if the method has no comments, or a lone &commat;inheritDoc.
+     *
+     * @return true if there are no comments, false otherwise
+     */
+    public boolean isSimpleOverride() {

```

Comment:
I wonder if we might be able to find a better name for this member? according to the comment, it will return true if there are no comments or there's an `inheritDoc` ... however in usage, it seems this is used to determine when docs should be inherited or not.

So perhaps it might be better to name it with the purpose ... the actual signal that it sends ... something like `public boolean shouldInheritDocs()` ?

Reply:
Make sense, will re-think for better way to describe following 2 status.

`isSimpleOverride()` -> No comment for this method.

`hasInheritDocTag` -> There is `{@inheritDoc}` tag, but there may be its own comments as well.